src/github/repository.ts

```diff
@@ -49,6 +51,10 @@ export const deleteRepositoryWebhookHandler = async (context: WebhookContext, ji
 		transformRepositoryId(context.payload.repository?.id, context.gitHubAppConfig?.gitHubBaseUrl)
 	);
 
+	// Update the total number of repos, we can assume atleast 1 repo(just deleted repo) exists for the default value
+	const totalNumberOfRepos = ((subscription.totalNumberOfRepos || 1) - 1);
```

Comment:
There's a `numberOfSyncedRepos` as well. Maybe should do a count like `select from RepoSyncStates`, as the repo we delete we can't be sure it is success or not?